imagedephi/rules.py

```diff
@@ -9,10 +9,13 @@ class FileFormat(Enum):
     SVS = "svs"
 
 
+expected_type_map: dict[str, list[Type[Any]]] = {"number": [int, float], "text": [str]}
```

Comment:
Oh that's interesting. Are these Philips fields part of XML? Could there be any number of values or do these fields have an expected count? I think we could still use these type checking rules in this case, but the implementation of the type checking would look different from how it looks here. Maybe it would use regex, maybe not.